daemon/main.go

```diff
@@ -639,6 +658,12 @@ func initEnv(cmd *cobra.Command) {
 
 	policy.SetPolicyEnabled(strings.ToLower(viper.GetString("enable-policy")))
 
+	for k, lbl := range fixedIdentity {
+		ni, _ := identity.ParseNumericIdentity(k)
+		identity.AddReservedNumericIdentity(ni, lbl)
+		identity.AddReservedIdentity(ni, lbl)
```

Comment:
This should be a single function being called here instead of encoding the identity cache internals here

Reply:
Done